src/player.cpp

```diff
@@ -27,7 +28,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "settings.h"
 #include "log.h"
 #include "porting.h"  // strlcpy
-
+#include "environment.h"
+#include "nodedef.h"
+#include "map.h"
+#include "collision.h"
```

Comment:
Needs sorting afterwards

Reply:
They weren't particularly sorted before, but I sorted all of those now (and it still compiles). How's this?